src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -1004,10 +1015,17 @@ function isJsonCell(value: ICellValue): boolean {
 	return !!(value && !value.isNull && value.displayValue?.match(IsJsonRegex));
 }
 
-function queryResultTextFormatter(showJsonAsLink: boolean, row: number | undefined, cell: any | undefined, value: ICellValue, columnDef: any | undefined, dataContext: any | undefined): string {
+function queryResultTextFormatter(showJsonAsLink: boolean, row: number | undefined, cell: any | undefined, value: ICellValue, columnDef: any | undefined, dataContext: any | undefined): string | { text: string, addClasses: string } {
 	if (showJsonAsLink && isJsonCell(value)) {
 		return hyperLinkFormatter(row, cell, value, columnDef, dataContext);
 	} else {
-		return textFormatter(row, cell, value, columnDef, dataContext);
+		const textFormatResult = textFormatter(row, cell, value, columnDef, dataContext);
+		if (DBCellValue.isDBCellValue(value) && value.isNull) {
```

Comment:
maybe you should update textFormatter to accept additional classes.

Reply:
updated